docs/openapi.yaml

```diff
@@ -0,0 +1,399 @@
+openapi: 3.0.0
```

Comment:
File doesn't validate nicely on https://editor.swagger.io .

I fixed some errors already (plural responses instead of singular response, or reponse ;) ). Some errors still left, mostly regarding missing references.